src/components/FormDropdownSearch/FormDropdownSearch.vue

```diff
@@ -76,6 +76,10 @@ const props = defineProps({
         type: Boolean,
         default: true,
     },
+    disableCache: {
```

Comment:
Maybe a little more specific?
```suggestion
    disableSearchCache: {
```